Project/package.json

```diff
@@ -1,33 +1,40 @@
 {
-  "name": "ACSCallingSample",
+  "name": "client",
   "version": "0.3.0",
   "private": true,
   "dependencies": {
-    "@azure/communication-administration": "1.0.0-beta.4",
-    "@azure/communication-calling": "1.6.2-beta.2",
-    "@fluentui/react": "^7.158.1",
+    "@azure/communication-administration": "^1.0.0-beta.4",
+    "@azure/communication-calling": "^1.7.0-beta.1",
+    "@fluentui/react": "^8.86.3",
+    "@uifabric/fluent-theme": "^7.7.7",
+    "@uifabric/icons": "^7.7.4",
     "react": "^16.14.0",
-    "react-dom": "^16.14.0"
+    "react-dom": "^16.14.0",
+    "uuid": "^8.3.2"
   },
   "scripts": {
-    "start": "webpack-dev-server --port 5000 --mode development",
-    "build": "webpack --mode development"
+    "build": "npm run clean && webpack --mode=production --env production",
```

Comment:
You will need to be able to also run this locally(Make sure you test locally and remote). I would expect to also see scripts here for building locally. 

Also, make sure when building locally it can still refresh local app when making local changes in the app code

Reply:
The scripts that were there before for building locally were:
"start": "webpack-dev-server --port 5000 --mode development",
"build": "webpack --mode development"
    
maybe you can name these "start-local" and "build-local" or as needed.

Reply:
Usually "build" is used for dev mode buildling. Id leave build for dev and do build-prod for prod/deplyment